Documentation/profiles.md

```diff
@@ -53,7 +53,17 @@ TODO
 
 ### Docker with User Namespaces
 
-TODO
+This profile allows user to start container with support of user namespace. The
+user on the host is `dockremap` which is automatically created by the docker
+daemon. All the containers started with user namespace enabled will have same
+`UID`, with following mapping:
```

Comment:
We should mention the GID too.

Also, other differences with the "default-docker" profile:
-  the processes in the container run as root of the user namespace of containers.
- "setgroups" is set to "deny" (check with `/proc/self/setgroups`, see `man user_namespaces`)
- Most other namespace kinds (cgroup, ipc, mnt, pid, uts) are owned by the new user namespaces, but the net namespaces is owned by the root user namespace. (is it actually true? I don't know how Docker configures this; we should check if that's the case. `man 2 ioctl_ns` explains how to test this with `ns_show`).